pagopa_it/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: pagopa-it-certificate-check
+description: Check the pagopa.it TLS/SSL certificate expiration date
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+dependencies:
+- name: microservice-chart
+  version: 1.17.0
```

Comment:
Can you use the version `1.20`?

Reply:
This is temporary. I'm going to update the template and I'll have to use a version above `1.20`

Reply:
We need to handle the image pull with secrets

Reply:
OK